src/layouts/default.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="wrapper">
-    <rd-header class="header" />
+    <rd-header class="header" @open-preferences="openPreferences" />
```

Comment:
Do we still want this button when `PREFERENCES` isn't 1?

Reply:
This is the action we take when the `open-preferences` event is emitted. I followed up with a change that adds a guard to the action to toggle this method if the feature flag isn't set.